File: my-folder/2727-number-of-senior-citizens/solution.c
int countSeniors(char ** details, int detailsSize){
    int result = 0;
    for (size_t i = 0; i < detailsSize; i++) {
        result += (details[i][11] > '6' || (details[i][11] == '6' && details[i][12] > '0')) ? 1 : 0;
    }
    return result;
}

File: my-folder/2727-number-of-senior-citizens/solution.rs
impl Solution {
    pub fn count_seniors(details: Vec<String>) -> i32 {
        details.iter()
            .filter(|detail| {
                let age_str = &detail[11..=12];
                age_str.parse::<u8>().unwrap() > 60
            })
            .count() as i32
    }
}
